deploy/tests/test_unittest_example.py

```
@@ -0,0 +1,9 @@
+import unittest
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)